app/src/main/res/xml/privacy_security_settings.xml

```diff
@@ -7,6 +7,11 @@
         android:layout="@layout/focus_preference"
         android:summary="@string/preference_privacy_and_security_description"/> -->
 
+    <org.mozilla.focus.settings.privacy.PreferenceToolTipCompose
+        android:key="@string/pref_key_tool_tip"
+        android:layout="@layout/focus_preference_compose_layout"
+        android:selectable="false"/>
+
```

Comment:
I do not see a separator in the Figma file, so we should use ` app:allowDividerBelow="false" ` here

Reply:
Done